src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -46,6 +46,24 @@ def dialog_content
       )
     end
 
+    def help_handler
+      Yast::Popup.LongText(
+        _("Help"),
+        RichText(
+          _("<p>The status of an alternative, can be automatic or manual. " \
```

Comment:
I think first sentence should be explanation what is alternative and why you would like to modify it. Then you can explain possible status. Consider, that user open module, say just WTF :question: and click on help to get idea.